x-pack/plugins/ml/public/application/jobs/jobs_list/components/job_details/format_values.js

```diff
@@ -63,7 +67,16 @@ export function formatValues([key, value]) {
     // numbers rounded to 3 decimal places
     case 'average_search_time_per_bucket_ms':
     case 'exponential_average_search_time_per_hour_ms':
-      value = typeof value === 'number' ? roundToDecimalPlace(value, 3).toLocaleString() : value;
+      value = roundTo3DecimalPlace(value);
```

Comment:
all of these conditions are the same, so they can all fall though to one condition as is happening on the lines above (68 + 69)
And with that change, the function `roundTo3DecimalPlace` probably isn't needed.